src/client/serialize/daos_serialize.c

```diff
@@ -261,6 +342,12 @@ daos_cont_serialize_props(hid_t file_id, daos_prop_t *prop_query)
 			if (rc != 0) {
 				D_GOTO(out, rc);
 			}
+        } else if (type == DAOS_PROP_CO_ROOTS) { 
```

Comment:
(style)  trailing whitespace
(style)  code indent should use tabs where possible
(style)  suspect code indent for conditional statements (8, 12)